.env.example

```
@@ -50,6 +50,10 @@ RAKAZO_LOCAL_MODELS=
 RAKAZO_LOCAL_MODELS_URL=http://127.0.0.1:11434/v1
 RAKAZO_LOCAL_CONTEXT_WINDOW=32768
 RAKAZO_LOCAL_MAX_TOKENS=4096
+# Comma-separated model ids on the local / openai-compatible endpoints that accept images.
+# Declared ids get input ["text","image"] so screenshot computer tools stay available.
+RAKAZO_LOCAL_VISION_MODELS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RAKAZO_LOCAL_VISION_MODELS key should go before the SANDBOX_COMMAND_TIMEOUT_MS key

(UnorderedKey)